snowflake/Makefile

```diff
@@ -5,6 +5,7 @@ all: bin
 
 .PHONY: bin
 bin:
+	make -C udfs/udfs/
```

Comment:
IMO it doesn't make sense to have this in the `bin` target
You should define a new target and add it as a requirement for the `all` target so that it is built by default.

Reply:
Right, thanks for pointing out.